themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/theme.scss

```diff
@@ -879,6 +879,98 @@ nav {
   }
 }
 
+// donate page selector
+
+.donate-type-intro {
+  font-size: 16px;
```

Comment:
I recommend switching all `font-size`s and `line-height`s to em or rem for accessibility 

https://css-tricks.com/is-it-better-to-use-ems-rems-than-px-for-font-size/

Reply:
Will do.

Reply:
There's more font-sizes on lines 917, 925, 934, and 955. There's line-heights on lines 918, 926, and 935.

Reply:
After you've updated those I think it's good to merge with master.

Reply:
Thanks for the suggestion, but there are numerous uses of px font sizes in this theme, so (a) this is arguably more consistent, (b) it's not within the scope of my task to update the whole theme's font-size convention, and (c) this theme will not be used after a few days from now anyway. Plus why change what is already in production. Merging now.